plugins/outputs/dynatrace/README.md

```diff
@@ -1,53 +1,120 @@
+'Ingest metrics' scope enabled. 
 
-* Managed https://{your-domain}/e/{your-environment-id}/api/v2/metrics/ingest
-* SaaS https://{your-environment-id}.live.dynatrace.com/api/v2/metrics/ingest
+The endpoint for the Dynatrace Metrics API v2 is 
 
-You can learn more about how to use the Dynatrace API [here](https://www.dynatrace.com/support/help/dynatrace-api/)
+* Managed `https://{your-domain}/e/{your-environment-id}/api/v2/metrics/ingest`
+* SaaS `https://{your-environment-id}.live.dynatrace.com/api/v2/metrics/ingest`
 
-You will also need to configure an API token for secure access. Find out how to create a token [here](https://www.dynatrace.com/support/help/dynatrace-api/environment-api/tokens/) or simply navigate to **Settings > Integration > Dynatrace API** in your Dynatrace environment and create a token with Dynatrace API and create a new token with 
-'Ingest metrics data points' access scope enabled. 
+```toml
+[[outputs.dynatrace]]
+  ## If no OneAgent is running on the host, url and api_token should be set
+
+  ## Dynatrace Metrics Ingest v2 endpoint to receive metrics
+  url = "https://YOUR_DOMAIN/api/v2/metrics/ingest"
+
+  ## API token is required if a URL is specified and should be restricted to the 'Ingest metrics' scope
+  api_token = "your API token here"
+```
+
+You can learn more about how to use the Dynatrace API [here](https://www.dynatrace.com/support/help/dynatrace-api/)
 
 ## Configuration
 
+### `url`
+
+*required*: `false`
+*default*: Local OneAgent endpoint
+
+Set Dynatrace environment URL (e.g.: `https://YOUR_DOMAIN/api/v2/metrics/ingest`) if you do not use a OneAgent or wish to export metrics directly to a Dynatrace metrics v2 endpoint. If a URL is set to anything other than the local OneAgent endpoint, then an API token is required.
+
 ```toml
-[[outputs.dynatrace]]
-  ## Leave empty or use the local ingest endpoint of your OneAgent monitored host (e.g.: http://127.0.0.1:14499/metrics/ingest).
-  ## Set Dynatrace environment URL (e.g.: https://YOUR_DOMAIN/api/v2/metrics/ingest) if you do not use a OneAgent
-  url = ""
-  api_token = ""
-  ## Optional prefix for metric names (e.g.: "telegraf.")
-  prefix = "telegraf."
-  ## Flag for skipping the tls certificate check, just for testing purposes, should be false by default
-  insecure_skip_verify = false
-  ## If you want to convert values represented as gauges to counters, add the metric names here
-  additional_counters = [ ]
-  ## Default dimensions will be added to every exported metric
-  default_dimensions = {
-    # key = "value"
-  }
+url = "https://{your-environment-id}.live.dynatrace.com/api/v2/metrics/ingest"
+```
+
+### `api_token` - require
 
+*required*: `false` unless `url` is specified
+
+API token is required if a URL is specified and should be restricted to the 'Ingest metrics' scope
+
+```toml
+api_token = "your API token here"
 ```
 
-## Requirements
+### `prefix`
+
+*required*: `false`
+
+```toml
+prefix = "telegraf"
```

Comment:
In the diff the prefix had a trailing `.` before - do we need to add it?

```suggestion
prefix = "telegraf."
```

Reply:
No